styles/styles.css

```diff
@@ -318,6 +318,138 @@
   color: #383838;
 }
 
+.sign-up--mob {
+  display: none;
+}
+
+.destinations {
+  display: flex;
+  flex-direction: column;
+  padding: 120px var(--col-width);
+}
+
+.destinations--title {
+  margin: 0 auto;
+  padding-bottom: 104px;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 47px;
+
+  color: #383838;
+}
+
+.destinations--buttons {
+  width: 623px;
+  height: 64px;
+
+  display: flex;
+  background-color: #F3F3F4;
+  border-radius: 8px;
+}
+
+.destinations--button {
+  width: 33.33%;
+  height: 64px;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+
+  color: #383838;
+  background-color: #F3F3F4;
+  border-radius: 8px;
+  border: 0;
+}
+
+.destinations--button:focus {
+  color: #FFFFFF;
+  background-color: #3077C6;;
+}
+
+.destinations--pictures {
+  padding-top: 64px;
+  padding-bottom: 49px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.destinations--picture--block {
+  height: 352px;
+  display: flex;
+  flex-direction: column;
+
+  justify-content: space-between;
+}
+
+.destinations--picture {
+  width: 296px;
+  height: 296px;
+}
+
+.destinations--name {
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+
+  color: #3077C6;
+}
+
+.destinations--button--down--block {
+  margin: 0 auto
+}
+
+.destinations--button--down {
+  border-radius: 50%;
+  border: 0;
+  height: 40px;
+  width: 40px;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.sign-up--web {
```

Comment:
please rearrange properies in your classes. First places for positions (fisplay, postion, aligns), next sizes (width, height),  gaps( margin, padding) and others